docs/pages/versions/unversioned/sdk/av.md

```diff
@@ -315,7 +315,7 @@ This default initial status can be overwritten by setting the optional `initialS
 
 When asked to seek an A/V item, native player in iOS sometimes may seek to a slightly different time. This technique, mentioned in [Apple documentation](https://developer.apple.com/documentation/avfoundation/avplayer/1387741-seek#discussion), is used to shorten the time of the `seekTo` call (the player may decide to play immediately from a different time than requested, instead of decoding the exact requested part and playing it with the decoding delay).
 
-If you matter about the precision more than about the delay, you can specify the tolerance with which the player will seek according to your needs.
+If you care about the precision more than about the delay, you can specify the tolerance with which the player will seek according to your needs.
```

Comment:
```suggestion
If precision is important, you can specify the tolerance with which the player will seek according to your needs. However, this will result in an increased delay.
```